releasenotes/notes/0.25/deprecate-instruction-qasm-9380f721e7bdaf6b.yaml

```
@@ -1,6 +1,6 @@
 ---
-deprecations:
+deprecations_circuits:
   - |
     The method :meth:`~qiskit.circuit.Instruction.qasm` and all overriding methods of subclasses
-    of `:meth:`~qiskit.circuit.Instruction` are deprecated. There is no replacement for generating
+    of `:class:~qiskit.circuit.Instruction` are deprecated. There is no replacement for generating
     an OpenQASM 2 string for an isolated instruction.
```

I would put a little bit more here like:

```suggestion
    an OpenQASM 2 string for an isolated instruction as typically
    a single instruction object has insufficient context to completely
    generate a valid OpenQASM2 string. If you're relying on this
    method currently you'll have to instead rely on the OpenQASM2
    exporter:  :meth:`.QuantumCircuit.qasm` to generate the OpenQASM2
    for an entire circuit object.
```

Mostly because I think the burden is a bit higher to explain deprecation for things were there is no alternative so there is a justification for users about the change.